injector/injector_test.go

```diff
@@ -142,7 +142,7 @@ func TestInjectTracker(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got := ti.InjectTracker(tt.args.vastXML, tt.args.NURL)
+			got, _ := ti.InjectTracker(tt.args.vastXML, tt.args.NURL)
```

Comment:
Now that we're returning the error from `InjectTracker`, what do you think about adding a test case to assert errors are being thrown correctly? I wouldn't think you need to check all cases, but the one's like "Invalid XML" or "XML Processing Error" maybe?

Reply:
@AlexBVolcy  Done.